File: transcendence/static/game/scripts/Vector2.py
class Vector2:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def add(self, x, y):
        self.x += x
        self.y += y

    def __str__(self):
        return f"Vector2({self.x}, {self.y})"

File: transcendence/static/game/scripts/Vector2.js
class Vector2 {
    constructor(x, y) {
        this.x = x;
        this.y = y
    }

    add(rightSideVector) {
        this.x += rightSideVector.x
        this.y += rightSideVector.y
    }

    toString() {
        return `Vector2(${this.x}, ${this.y})`
    }
}